src/skmultiflow/drift_detection/ddm.py

```diff
@@ -61,17 +61,19 @@ class DDM(BaseDriftDetector):
     >>> ddm = DDM()
     >>> # Simulating a data stream as a normal distribution of 1's and 0's
     >>> data_stream = np.random.randint(2, size=2000)
-    >>> # Changing the data concept from index 999 to 1500, simulating an 
+    >>> # Changing the data concept from index 999 to 1500, simulating an
     >>> # increase in error rate
     >>> for i in range(999, 1500):
     ...     data_stream[i] = 0
     >>> # Adding stream elements to DDM and verifying if drift occurred
     >>> for i in range(2000):
     ...     ddm.add_element(data_stream[i])
     ...     if ddm.detected_warning_zone():
-    ...         print('Warning zone has been detected in data: ' + str(data_stream[i]) + ' - of index: ' + str(i))
+    ...         print("Warning zone has been detected in data: {} - of index: {}"
+    ...                 .format(data_stream[i],i))
```

Comment:
The number of spaces before the the `.format` statement seems to be incorrect. Nonetheless, when I try to create a suggestion, the indentation looks correct in the code editing tool. Visually, the format command should be aligned with the start of the string. Perhaps the rendering process is making it look unaligned. Any thoughts @garawalid?vvvvvvvvvvv

Reply:
In fact, it's me who forced the format to be in the second line, so I can respect the max length rule. In the IDE, it's working like a charm. I'll change it as you suggested.

Reply:
Thanks for addressing that!